packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/telecom-telephony-alias-portability-portabilities.routing.js

```diff
@@ -0,0 +1,28 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('telecom.telephony.alias.portabilities', {
+    url: '/portabilities',
+    views: {
+      'aliasView@telecom.telephony.alias': 'portabilities',
+    },
+    translations: { value: ['.'], format: 'json' },
+    resolve: {
+      billingAccount: /* @ngInject */ $transition$ => $transition$.params().billingAccount,
+      attachMandate: /* @ngInject */ ($state, billingAccount) => portability => $state.go('telecom.telephony.alias.portabilities.attach', {
+        billingAccount,
+        portabilityId: portability.id,
+      }),
+      goToPortabilities: /* @ngInject */ ($state, billingAccount) => (message = false, type = 'success') => {
+        const reload = message && type === 'success';
+
+        const promise = $state.go('telecom.telephony.alias.portabilities', {
+          billingAccount,
+        },
+        {
+          reload,
+        });
+
+        return promise;
```

Comment:
You don't display the message to customer ?

Reply:
there is no message.